styles/style.css

```diff
@@ -459,6 +652,43 @@ input[type="submit"]:hover {
   height: 5px;
   position: absolute;
   bottom: 8px;
+  left: calc(50% - 68px);
   background: #0d151e;
   border-radius: 10px;
 }
+
+/* Desktop version of about me section */
+@media screen and (min-width: 768px) {
```

Comment:
[OPTIONAL] 🤓⭐ great job implementing the required changes. Optionally, I recommend you enclose all the desktop syles inside **ONE** media query statement. there is no need to write a media query for each section.